vermouth/processors/tune_idp_bonds.py

```diff
@@ -21,14 +21,34 @@
 
 from .processor import Processor
 from ..graph_utils import make_residue_graph
-from ..rcsu.go_utils import get_go_type_from_attributes, _get_resid_region, _in_resid_region
+from ..rcsu.go_utils import get_go_type_from_attributes, _in_resid_region
 import numpy as np
-from vermouth import selectors
+from ..selectors import select_backbone
 from ..log_helpers import StyleAdapter, get_logger
-
+from ..dssp import annotate_residues_from_sequence
 LOGGER = StyleAdapter(get_logger(__name__))
 
 
+def make_disorder_string(molecule, idr_regions):
+    """
+    make a string for annotating order and disorder along a molecule
+    """
+
+    order_disorder_seq = []
+
+    for key, node in molecule.nodes.items():
+        if select_backbone(node):
```

Comment:
I wouldn't tag just the  backbone (or SC1), but rather just all nodes with the appropriate resid